Makefile.am

```diff
@@ -21,15 +21,16 @@ distwin32-zip : distwin32-dir
 distwin32-dir :
 	rm -rf liblouis-$(VERSION)-win32/*
 	@if which i586-mingw32msvc-gcc >/dev/null; then \
+	  export CPPFLAGS='-I/root/build/win32/libyaml/include/' && \
+	  export LDFLAGS='-L/root/build/win32/libyaml/lib/' && \
```

Comment:
It doesn't really bug me that much, but this path seems just as arbitrary or magic as the `/usr/i686-w64-mingw32/...` path. At least the latter is somewhat standard.